testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -773,7 +773,7 @@ private void assertTestingMaterializedViewQuery(String schema, String materializ
 
     private void createTestingMaterializedView(QualifiedObjectName view, Optional<String> comment)
     {
-        assertUpdate(format("CREATE SCHEMA IF NOT EXISTS %s", view.getSchemaName()));
```

Comment:
would that work, if execued, and create schema with wrong location, or would that fail?

Reply:
Fails as by default it tries to create data in HDFS.

Reply:
awesome.

would it be possible to configure default warehouse location (on HMS side?), so that CREATE SCHEMA just works?
cc @findinpath

Reply:
I am assuming that in `protected QueryRunner createQueryRunner()` should be added something like:

```
        queryRunner.execute("CREATE SCHEMA " + SCHEMA + " WITH (location = 's3://" + bucketName + "/" + SCHEMA + "')");
```

Reply:
@findinpath It is like this. Please look at schema initialiser here https://github.com/trinodb/trino/blob/c2681d23ec23610b7d3a0daee24abccd1e33918a/plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergQueryRunner.java#L64
@findepi I assume we could but this would mean we will need to provide hive-site.xml for every use case separately and IMHO its less clear to others that overloaded method.
Config in hive-site.xml
```
hive.metastore.warehouse.dir
``` 
There is also such setting for Glue `hive.metastore.glue.default-warehouse-dir` we could implement something similar for HMS or made it a connector wide setting.